code/script.js

```diff
@@ -1,51 +1,482 @@
+                <button class="chat-link" onclick=" window.open('https://www.riksdagen.se/sv/ledamoter-partier/miljopartiet/','_blank')">The Green Party</button>
+    `), 4000);
+
+		setTimeout(() => botReply(`So will you become politician now?`), 10000);
+		setTimeout(() => delayHTML(), 11000);
+
+		const delayHTML = () => {
+			inputWrapper.innerHTML = `
+			<form class="politicsConatainer">
+				<button class="politicsBtn" type="button" Value="Yes, I will limit air traffic!">Yes, I will limit air traffic!</button>
+				<button class="politicsBtn" type="button" Value="No, it's not going to happen">No, it's not going to happen</button>
+			</form>
+		`;
+
+			document
+				.querySelectorAll(".politicsBtn")
+				.forEach((button) => button.addEventListener("click", () => nextQuestion(button.value)));
+		};
+
+	} else {
+	//If user "select your reason..." for continue as usual in previous question, show these messages:
+		botReply(`I see.`);
+		setTimeout(() => botReply(`The climate crisis affects many people's psychological wellbeing.`), 1000);
+		setTimeout(() => botReply(`Maybe this article could help you find some new perspectives.`), 3000);
+		setTimeout(() => botReply(`<button class="chat-link" onclick=" window.open('https://www.health.harvard.edu/blog/is-climate-change-keeping-you-up-at-night-you-may-have-climate-anxiety-202206132761/','_blank')">Harvard Health</button>`), 5000);
+
+		setTimeout(() => botReply(`Do you feel more enlightened now?`), 10000);
+
+		setTimeout(() => delayHTML(), 11000);
+		const delayHTML = () => {
+			inputWrapper.innerHTML = `
+			<form class="usefulConatainer">
+					<button class="usefulBtn" type="button" Value="Yes!">Yes!</button>
+					<button class="usefulBtn" type="button" Value="No.">No</button>
+			</form>
+		`;
+
+			document
+				.querySelectorAll(".usefulBtn")
+				.forEach((button) => button.addEventListener("click", () => anythingElse(button.value)));
+		};
+	}
+};
+
+// 7th question - Final question 
+const anythingElse = (anythingResponse) => {
+	botReply(`Is there anything else we can help you with?`);
+
+	setTimeout(() => delayHTML(), 1000);
+
+	const delayHTML = () => {
+		inputWrapper.innerHTML = `
+			<form id="anythingBtns" >
+				<button id="anythingYes" type="button">Yes</button>
+				<button id="anythingNo" type="button">No</button>
+			</form>
+		`;
+
+		document
+			.getElementById("anythingYes")
+			.addEventListener("click", () => rateAnxiety("Yes"));
+
+		document
+			.getElementById("anythingNo")
+			.addEventListener("click", () => thankYou("No"));
+	};
+};
+
+// Final message from bot with contact information
+const thankYou = (value) => {
+	botReply(`Thank you! Please, feel free to reach out to us if you have further questions`);
+
+	setTimeout(() => (inputWrapper.innerHTML = `
+			<form>
+  				<button id="contact" class="contactUs" type="email"><a href="mailto:email@email.de" taget="_blank">Contact us</a></button>
```

Comment:
Not sure if this is because of something I do wrong or if it is a bug, but the contact-button does not work every time.